dcos/marathon.py

```diff
@@ -54,6 +54,11 @@ def _to_exception(response):
     if isinstance(response, Exception):
         return DCOSException(_default_marathon_error(str(response)))
 
+    if response.status_code == 400:
```

Comment:
no, in `test_show_bad_app_version` the resulting error is from status_code `422`